projects/igniteui-angular/src/lib/directives/for-of/base.helper.component.ts

```diff
@@ -0,0 +1,42 @@
+import {
+    EventEmitter,
+    Output,
+    HostListener,
+    ElementRef,
+    ChangeDetectorRef,
+    OnDestroy
+} from '@angular/core';
+
+export abstract class VirtualHelperBase implements OnDestroy {
+    public scrollAmount = 0;
+
+    public _size = 0;
```

Comment:
Probably better to make it a concrete class with a directive decorator mainly because of loss of metadata in
v9 with Ivy